diff --git a/device/testing/linux-postmarketos-qcom-msm8996/APKBUILD b/device/testing/linux-postmarketos-qcom-msm8996/APKBUILD
index 601fa9f44e..7877d913ef 100644
--- a/device/testing/linux-postmarketos-qcom-msm8996/APKBUILD
+++ b/device/testing/linux-postmarketos-qcom-msm8996/APKBUILD
@@ -3,7 +3,7 @@ maintainer="Yassine Oudjana (Tooniis) <y.oudjana@protonmail.com>"
 _flavor="postmarketos-qcom-msm8996"
 pkgname=linux-$_flavor
 pkgver=6.19.5
-pkgrel=0
+pkgrel=1
 pkgdesc="Kernel close to mainline with extra patches for Qualcomm MSM8996 devices"
 arch="aarch64"
 _carch="arm64"
@@ -32,6 +32,7 @@ _tag=v${pkgver//_/-}-msm8996
 source="
 	linux-$_tag.tar.gz::$url/-/archive/$_tag/linux-$_tag.tar.gz
 	config-$_flavor.$arch
+	0001-add-samsung-gts3lwifi-dts.patch
 "
 builddir="$srcdir/linux-$_tag"
 
@@ -63,4 +64,5 @@ package() {
 sha512sums="
 dd30dd113ad9f0b4ab3771ea655de2becce7ddaf5066d2f860b7bb1eb78573137313191d9da740b152348f9f89894ace2bb94fb195e340f7b6dabedeebbc929f  linux-v6.19.5-msm8996.tar.gz
 6a1e3a18fb454378ef5569a6951426d118144985c9028e663f46d28cde6784723296601d3756176045e22d706a9195938804c6a82f02fcf548e18d2e66c35b27  config-postmarketos-qcom-msm8996.aarch64
+dfb0cba014d7ca85e3275b553774d217c0f38670832bb83e50cccdeeee7b1ccb2780a4501605edc5beb7ef7622611c885d1355cfdbc4f3cb7912721635df26cb  0001-add-samsung-gts3lwifi-dts.patch
 "
